srcpkgs/pam_zfscrypt/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'pam_zfscrypt'
+pkgname=pam_zfscrypt
+version=0.2.0
+revision=1
+wrksrc="zfscrypt-${version}"
+build_style=gnu-makefile
+make_use_env=yes
+hostmakedepends="zfs-devel"
+makedepends="pam-devel zfs-devel"
+make_dirs="/usr/lib/security 0755 root root"
+depends="zfs pam"
+short_desc="Automatically decrypt and mount encrypted ZFS home directories"
+maintainer="Zach Dykstra <dykstra.zachary@gmail.com>"
+license="MIT"
+homepage="https://github.com/BenKerry/zfscrypt"
+distfiles="https://github.com/BenKerry/zfscrypt/archive/v${version}.tar.gz"
+checksum=@032766fca0f4d6da47faa29a92195f8357f2e8073e49e14c9cb719b3248c3a88
+noverifyrdeps=yes
```

Comment:
Don't do this, add zfs's libs to common/shlibs 

    /usr/lib/libzfs.so.2 zfs-0.8.2_1
    /usr/lib/libnvpair.so.1 zfs-0.8.2_1

etc